### PR TITLE
fix: broken TestHardcodedIncludeDirectiveDDOS2 on windows

### PR DESCRIPTION
**Pull request by jabdr**
This is a small fix for the TestHardcodedIncludeDirectiveDDOS2 on windows.

Windows refuses to delete files as long as the file descriptor is open. Closing the temporary file fixes this issue and go test is able to cleanup the files.

### CODE REVIEW

File: internal/seclang/parser_test.go [hunk 1]
```diff
@@ -169,6 +169,7 @@ func TestHardcodedIncludeDirectiveDDOS2(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	tmpFile2.Close()
```
Comment (jptosso) [inline internal/seclang/parser_test.go:172]: shouldn't we assert errors on tmpFile2.Close()?

Comment (jabdr) [reply internal/seclang/parser_test.go]: Done

**Review by jcchavezs — CHANGES_REQUESTED** [review]
Thanks! I think it is way easier to create the file on before hand instead of relying on an existing unrelated file in the system. You can do `os.CreateTemp(t.TempDir(), "existing.txt")`. This could be merged right away after that.

**Comment by jabdr** [comment]
@jcchavezs  I am not sure what you mean. I do not want to touch the existing code in this PR, just fix the missing Close statement (which is also required with os.CreateTemp).

**Review by jcchavezs — APPROVED** [review]

**Pull request merged**